haiku-apps/romannumbers/romannumbers-1.0~git.recipe

```diff
@@ -10,9 +10,9 @@ HOMEPAGE="https://github.com/cnelmortimer/RomanNumbers"
 COPYRIGHT="2019 N.C. Cruz"
 LICENSE="MIT"
 REVISION="1"
```

Comment:
This number needs to be bumped.

Reply:
Fixed... but I finally forgot adding "--amend" when committing so the PR might be outdated. Do I need to fix that (how?) or the PR automatically goes to the latest commit?